lib/engine/game/g_18_sj/step/buy_sell_par_shares.rb

```diff
@@ -10,6 +10,22 @@ class BuySellParShares < Engine::Step::BuySellParShares
           def can_sell?(entity, bundle)
             super && (@game.oscarian_era || bundle.corporation.floated?)
           end
+
+          def can_ipo_any?(entity)
+            !bought? && @game.corporations.any? do |c|
+              @game.can_par?(c, entity) && can_buy?(entity, c.ipo_shares.first&.to_bundle)
+            end
+          end
+
+          # FIXME: copied from 21moon which has "FIXME: move to common location"
```

Comment:
can you fix it?

Reply:
I am asking Roseundy about that comment. It might be that 21moon has duplicate methods (in buy_sell_par_shares.rb and corporation_buy_sell_shares.rb) so that the comment is for 21moon, or it might be something to be shared more widely.

Reply:
@tobymao I asked Steven; the FIXME was 21moon specific. Do not see any special need to make a more generalized solution at this time.